src/Symfony/Component/Security/Http/Authentication/AuthenticatorManager.php

```diff
@@ -58,6 +58,7 @@ public function __construct(
         private bool $eraseCredentials = true,
         private bool $hideUserNotFoundExceptions = true,
         private array $requiredBadges = [],
+        private bool $showAccountStatusExceptions = false,
```

Comment:
should it rather be `hideAccountStatusExceptions` to be consistent with `hideUserNotFoundExceptions`?